packages/wds-event-tool/src/main/kotlin/org/webdevandsausages/events/service/RegistrationService.kt

```diff
@@ -45,7 +45,7 @@ class CreateRegistrationService(
                 )
                 else -> {
                     val event = eventData.t.event
-                    val numRegistered = eventData.t.participants.size
+                    val numRegistered = eventData.t.participants.filter { it.status == ParticipantStatus.REGISTERED }.size
```

Comment:
I noticed when testing that if user has cancelled before there is wait list present, it will be counted here, so wait list will start too soon because number of registered people contains also cancelled ones